src/components/Shipment-dialog.jsx

```diff
@@ -127,28 +126,43 @@ export default function CreateShipmentModal({
             Create New Shipment
           </DialogTitle>
         </DialogHeader>
-       
+
         <form onSubmit={handleSubmit} className="space-y-4 p-4">
           <p className="text-sm text-muted-foreground">
             Enter the details for the new shipment.
           </p>
           <div className="space-y-2">
-             <Label htmlFor="clientId">Select Available client</Label>
-              <Select
-                onValueChange={(value) => handleSelectChange("clientId", parseInt(value, 10))}
-              >
-                <SelectTrigger id="clientId">
-                  <SelectValue placeholder="Select a client" />
-                </SelectTrigger>
-                <SelectContent>
-                  {clients.map((prop) => (
-                    <SelectItem key={prop.clientId} value={prop.clientId}>
-                      {prop.clientName} (ID: {prop.clientId})
+            <Label htmlFor="driverId">Select Available Driver</Label>
```

Comment:
I believe the changes were made here, you've changed the logic to select the client to instead select available driver. Here we need to select the available clients.